RELEASE-NOTES.md

```diff
@@ -12,6 +12,7 @@ The project release numbers follow [Semantic Versioning](http://semver.org/spec/
 ## [Version 0.3.0] - Release date 2020-01-08
 
 ### Added
+- Added blt_assert_exists() utility macro.
```

Comment:
It looks like this note was added to the wrong section -- it should be under ``Unreleased``, right?